include/drivers/i2c.h

```diff
@@ -337,6 +337,166 @@ struct i2c_slave_config {
 	const struct i2c_slave_callbacks *callbacks;
 };
 
+#if defined(CONFIG_I2C_STATS) || defined(__DOXYGEN__)
```

Comment:
note: if all content below is internal, Doxygen may be dropped after all (no need to use `INTERNAL_HIDDEN`)